integration-tests/serverless/test-project/.env.yaml

```diff
@@ -1,4 +1,3 @@
 NODE_OPTIONS: "-r dd-trace/init"
-DD_TRACE_DEBUG: "true"
```

Comment:
since the mini agent now inherits all stdio from the parent, the tracer doesn't need to be set to the debug level to show mini agent logs (which are read for the integration test)